cli/tests/common/test_environment.rs

```diff
@@ -229,6 +229,16 @@ impl TestEnvironment {
             .push((path.into(), replacement.into()));
     }
 
+    /// Sets up echo as a merge tool.
+    ///
+    /// Windows machines may not have the echo executable installed.
+    pub fn set_up_fake_echo_merge_tool(&self) {
+        let echo_path = assert_cmd::cargo::cargo_bin!("fake-echo");
+        assert!(echo_path.is_file());
+        let echo_path = to_toml_value(echo_path.to_str().unwrap());
+        self.add_config(formatdoc!("merge-tools.echo.program = {echo_path}"));
```

Comment:
nit: rename to `merge-tools.fake-echo`

Reply:
Thanks. That's what I meant, and isn't different than hooking into other `fake-*` commands.